tests/tick.test.mjs

```diff
@@ -455,8 +474,31 @@ describe('Tick', async () => {
       timestamp: distributionTimestamp,
     });
 
-    // assert no error tag
-    assertNoResultError(distributionTick);
+    // assert multiple messages are sent given the tick notice, epoch created notice and epoch distribution notice
+    assert.equal(distributionTick.result.Messages.length, 3); // cannot explain why this is 4, i'd expect it to be 3 messages (1 tick notice, 1 epoch created notice, 1 epoch distribution notice)
+    // tick notice is sent
+    assert.equal(
+      distributionTick.result.Messages[0].Tags.find(
+        (tag) => tag.name === 'Action',
+      ).value,
+      'Epoch-Created-Notice',
+    );
+    // new epoch is created
+
+    // epoch distribution notice is sent
+    assert.equal(
+      distributionTick.result.Messages[1].Tags.find(
+        (tag) => tag.name === 'Action',
+      ).value,
+      'Epoch-Distribution-Notice',
+    );
+
+    assert.equal(
+      distributionTick.result.Messages[2].Tags.find(
+        (tag) => tag.name === 'Action',
+      ).value,
+      'Tick-Notice',
+    );
```

Comment:
validates all three messages get sent when a new epoch is created and distributed